app/src/main/res/values-es/strings.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">Warpinator</string>
```

Comment:
It is really bad practice to do that. Untranslatable strings should definitely not have copies in other files and that's why I removed them.

Reply:
I don't see much wrong with it. Does it cause some undefined behavior when it is left there? Now I'd have to nag every translator who barely knows how git functions to remove it from their PRs to keep consistence with the other files. This is IMO more hassle than spotting that someone might have accidentally changed an untranslatable string.

Reply:
It shouldn't cause undefined behaviour in this case, in others, as of my understanding, it could. And also, if you want to rename the app, you would have to change it in every file.

But will discuss this later in a translation PR